### PR TITLE
adding filtering for products

### LINKED ISSUE #9: Add search function (discount and used)
* Search bar needs to be able to search fpr discounted and used products

### PR DESCRIPTION
**Pull request by Transurgeon**
this PR will eventually take care of issue #14 .

But for now it only implements the changes made by Anas on the frontend.

The filtering of products is done for now.

resolves #9

### CODE REVIEW

File: WebContent/index.jsp
Comment (cedparadis) [inline WebContent/index.jsp:1]: on line 47 the methods getAllProducts(); is not modified. We only modified getProductDetails();
Just copy the sql statements from product details to all products

Comment (Transurgeon) [reply WebContent/index.jsp:1]: ok done. thanks for the pointer

File: WebContent/userHome.jsp [hunk 1]
```diff
@@ -18,36 +18,53 @@
...
 	/* Checking the user credentials */
 	String userName = (String) session.getAttribute("username");
 	String password = (String) session.getAttribute("password");
+	String userType = (String) session.getAttribute("usertype");
 
-	if (userName == null || password == null) {
+	boolean isValidUser = true;
 
-		response.sendRedirect("login.jsp?message=Session Expired, Login Again!!");
+	if (userType == null || userName == null || password == null || !userType.equals("customer")) {
```
Comment (cedparadis) [inline WebContent/userHome.jsp:29]: change "customer" to "student"

**Review by cedparadis — APPROVED** [review]
Looks very good! nice work, just change that one word in my comment, but still approving the request

**Pull request merged**